AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -123,6 +123,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 th-deg.de#$#body { overflow: visible!important; }
 th-deg.de#$#div[id^="CybotCookiebotDialog"] { display: none!important; }
 rohm.co.jp###showcookiealert
+mobonline.com.br##body .x-lgpd-popup
```

Comment:
Можно так
`mobonline.com.br##body .x-lgpd-popup.is--active`

Reply:
If I'm not wrong, using `:not(whatever)` also should works, for example:
```
mobonline.com.br##.x-lgpd-popup:not(#style_important)
```
because it looks like that it have higher priority.
I don't say that it's preferable, but I think it may be useful in some cases.